### PR TITLE
fix: improve build-docs command

### PR DESCRIPTION
**Pull request by AlexVarchuk**
## What/Why/How?

- fix `template.hbs` problem (we do not include `template.hbs` in publish flow for `build-docs` command. This should fix it.)
- add `apis` to build-docs
- rename `options` to `features.openapi`
## Screenshots (optional)
**BEFORE**
Screenshot 2022-09-21 at 17 09 10

**AFTER**
Screenshot 2022-09-21 at 17 07 42

## Check yourself

- [x] Code is linted
- [x] Tested with redoc/reference-docs/workflows
- [x] All new/updated code is covered with tests

## Security

- [x] Security impact of change has been considered
- [x] Code follows company security practices and guidelines

### CODE REVIEW

File: packages/cli/src/index.ts [hunk 1]
```diff
@@ -362,53 +362,53 @@ yargs
...
+          t: {
+            alias: 'template',
+            describe: 'Path to handlebars page template, see https://git.io/vh8fP for the example',
+            type: 'string',
+          },
+          templateOptions: {
+            describe:
+              'Additional options that you want pass to template. Use dot notation, e.g. templateOptions.metaDescription',
+          },
+          features: {
```
Comment (adamaltman) [inline packages/cli/src/index.ts:398]: Isn't it features.openapi here?

File: packages/cli/src/commands/build-docs/types.ts [hunk 2]
```diff
@@ -19,5 +19,8 @@ export type BuildDocsArgv = {
   disableGoogleFont?: boolean;
   template?: string;
   templateOptions: Record<string, any>;
-  options: string | Record<string, unknown>;
+  features: {
+    openapi: string | Record<string, unknown>;
+  };
```
Comment (adamaltman) [inline packages/cli/src/commands/build-docs/types.ts:24]: Isn't `features.openapi` the single string? Not an object in an object?

File: packages/cli/src/index.ts (hunk 1 above)
Comment (AlexVarchuk) [reply packages/cli/src/index.ts:398]: according to the existing [issue](https://github.com/yargs/yargs/issues/1230) and this [comment](https://github.com/yargs/yargs/issues/995#issuecomment-352326378) `yargs` do not support straight forward solution for this `dot-notation`. It is a workaround solution for that.

File: packages/cli/src/commands/build-docs/types.ts (hunk 2 above)
Comment (AlexVarchuk) [reply packages/cli/src/commands/build-docs/types.ts:24]: No, it workaround flow for support `dot-notation`.  I tried update to the latest version of yargs, but they still have this issue.
```bash
redocly build-docs -- --config=./resources/.redocly.yaml --features.openapi.theme.colors.primary.main=red
```
Screenshot 2022-09-26 at 14 48 55

**Review by tatomyr — APPROVED** [review]

**Pull request merged**